src/main/java/com/jvm_bloggers/entities/newsletter_issue/NewsletterIssueRepository.java

```diff
@@ -18,4 +20,8 @@
 
     List<NewsletterIssue> findAllByOrderByPublishedDateDesc();
 
+    @Query("select case when count(*) > 0 then true else false end "
+        + "from NewsletterIssue where issueNumber = :issueNumber")
+    boolean issueNumberExist(@Param("issueNumber") Long issueNumber);
```

Comment:
@airborn I was considering using existing method `Option<NewsletterIssue> findByIssueNumber(Long issueNumber)` but wanted to avoid overhead related to eagerly loading some of the relations to `NewsletterIssue` like (`BlogPost`s and `Blog`s) when I just want to check if given `issueNumber` exist in DB or not. When visiting `jvm-bloggers` home page there will be 3 calls to load whole `NewsletterIsssue`. 

With above I only wanted to justify why I took this approach and if using `Option<NewsletterIssue> findByIssueNumber(Long issueNumber)` is a preferred way then I will make the appropriate change in PR.